asdf_astropy/converters/wcs/fitswcs.py

```diff
@@ -0,0 +1,19 @@
+from asdf.extension import Converter
+
+
+class FitsWCSConverter(Converter):
+    tags = ("tag:astropy.org:astropy/fits/fitswcs-*",)
+    types = ("astropy.wcs.wcs.WCS",)
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from astropy.wcs import WCS
+
+        return WCS(node["hdu"][0].header, fobj=node["hdu"])
+
+    def to_yaml_tree(self, wcs, tag, ctx):
+        node = {}
+        if wcs.sip is not None:
```

Comment:
Are you referring to the "table" and "TAB" distortions or is there another type?
- "TAB" is a non-starter since astropy can't write them: https://github.com/astropy/astropy/issues/9998
- "table" is tested: https://github.com/astropy/asdf-astropy/pull/246/files#diff-4d12199a58487bbc9c9a729bcb6f8a3d6297a6547eb1510c29c6a8721986b43eR40